folding/validators/protein.py

```diff
@@ -221,6 +221,12 @@ def setup_simulation(self):
 
         self.pdb_complexity = Protein._get_pdb_complexity(self.pdb_location)
         self.init_energy = self.calc_init_energy()
+        # Checking if init energy is nan
+        if self.init_energy != self.init_energy:
```

Comment:
This does work as a check for nan, I prefer this way but if we want it to be more readable we can just do math.isnan()